client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -265,16 +265,19 @@ export default () => {
             </Box>
             <Typography variant='h2'>Participant Details</Typography>
             <Grid container spacing={2} className={classes.gridSection}>
-              {Object.keys(keyLabelMap).map((key) => (
-                <Grid key={key} item xs={12} sm={6} xl={3}>
-                  <Typography variant='body1'>
-                    <b>{keyLabelMap[key]}</b>
-                  </Typography>
-                  <Typography test-id={'participantDetailsView' + key} variant='body1'>
-                    {participant[key]}
-                  </Typography>
-                </Grid>
-              ))}
+              {Object.keys(keyLabelMap).map(
+                (key) =>
+                  participant[key] && (
+                    <Grid key={key} item xs={12} sm={6} xl={3}>
```

Comment:
Do we still want to view the label regardless if the value is null/undefined?

Reply:
Potentially, although I feel like it's cleaner to just hide it. @fw-noel, thoughts?

Reply:
Alternative idea: put the site name right in the status string? So it would say something like "Available" or "Hired - Sleepy Cat Hospital" for status.

Reply:
Hiding it I think is the way to go